module_1/src/App.js

```diff
@@ -0,0 +1,25 @@
+
+import './App.css';
+import Counter from "./components/Counter.js";
+import SearchForm from './components/SearchForm.js';
+import GenreSelect from './components/GenreSelect.js';
+
+function App() {
+  var searchFormProps = {};
+  searchFormProps.initialQuery = "horror";
+  searchFormProps.onSearch = (query) => {console.log("search initiated with query: " + query)}
```

Comment:
Good solution. Would be even nicer/simpler to write
```
const searchFormProps = {
  initialQuery: "horror",
  onSearch: (query) => { console.log("search initiated with query: ", query); }
}
```

Please try to avoid using `var` and use `const` or `let` instead.
In console.log it's a syntactic sugar to write the variable after a comma.